test/models/article_test.rb

```diff
@@ -13,6 +13,14 @@ class ArticleTest < ActiveSupport::TestCase
     assert_equal '/ogp/ruby_on_rails.png', article.selected_thumbnail_url
   end
 
+  test '#published?' do
+    public_article = articles(:article1)
+    wip_article = articles(:article3)
+
+    assert public_article.published?
+    assert_not wip_article.published?
```

Comment:
```suggestion
    assert articles(:article1).published?
    assert_not articles(:article3).published?
```

どちらがいいか微妙かもですが、プロジェクト全体の書き方の具合からしてこのような感じに略しちゃうのがいいかもです。